### PR TITLE
css

### PR DESCRIPTION
**Pull request by Kiran2821996**

### CODE REVIEW

File: src/components/trending/TrendItems.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ function TrendItems({ data }) {
     <div>
       {data.map((item, idx) => {
         return (
-          <div className="trend_block_item" key={idx}>
+          <div className="trend_block_item" key={idx} onClick>
```
Comment (Souvikdeb2612) [inline src/components/trending/TrendItems.jsx]: What is the onclick here?

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/trending/TrendItems.jsx [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+import React, { useState } from "react";
 
 function TrendItems({ data }) {
+  const [dataClick,setdataClick]=useState("");
+  const [toggle,setToggle]=useState(false)
+
+  const handleClick=(item)=>{
+setdataClick(item)
+setToggle(true)
+console.log(dataClick)
```
Comment (Souvikdeb2612) [inline src/components/trending/TrendItems.jsx]: remove console statements before raising pr

File: src/components/trending/TrendItems.jsx [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+  }
+  const handleClose=()=>{
+    console.log("llll")
```
Comment (Souvikdeb2612) [inline src/components/trending/TrendItems.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/trending/TrendItems.jsx (hunk 3 above)
Comment (Kiran2821996) [reply src/components/trending/TrendItems.jsx]: done

File: src/components/trending/TrendItems.jsx (hunk 2 above)
Comment (Kiran2821996) [reply src/components/trending/TrendItems.jsx]: done

File: src/components/trending/TrendItems.jsx (hunk 1 above)
Comment (Kiran2821996) [reply src/components/trending/TrendItems.jsx]: done sie

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**